ui/redux/actions/sync.js

```diff
@@ -0,0 +1,343 @@
+import * as ACTIONS from 'constants/action_types';
```

Comment:
Did you make any changes to this file?

Reply:
https://github.com/lbryio/lbry-redux/pull/355/commits/79f67889831a986b4f8a7b629c776b725f9614d2 here's a handy diff

Reply:
I brought the stuff from syncwrapper in.

Reply:
https://github.com/lbryio/lbry-desktop/pull/4882/commits/19e11742fb3ef92f4ab6a987052c10f6e6bac937